### ISSUE TITLE
ccv2 plugin may fail parsing manifest.json

### ISSUE THREAD

**Issue by mpern** [sap-commerce-tools/commerce-gradle-plugin#9]
If you the project uses a very minimal manifest (without `storefrontAddons`, `properties` or `aspects`) applying the plugin will fail with NPEs

**Workaround** (for now):

add empty arrays for any of the above keys

```
{
    "commerceSuiteVersion": "1811.3",
    "extensions": [
        ...
    ],
    "storefrontAddons": [],
    "properties": [],
    "aspects": []
}
```

**Comment by mpern** [comment]
fixed in `v1.3.1`

**Comment by mpern** [comment]
https://github.com/sap-commerce-tools/commerce-gradle-plugin/releases/tag/v1.3.1
